ThunderCloud/StormLanguageController.swift

```diff
@@ -0,0 +1,410 @@
+//
+//  StormLanguageController.swift
+//  ThunderCloud
+//
+//  Created by Matthew Cheetham on 02/06/2017.
+//  Copyright © 2017 threesidedcube. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+@objc(TSCStormLanguageController)
+/// A subclass of ThunderBasic's `TSCLanguageController` which adds extra properties and methods to the controller more useful to Storm's needs
```

Comment:
Would be good to get some top-level documentation on how storm strings are formatted differently to the strings `Locale` uses (hence why you have to flip the components of the string), and the difference between a Language and a Region e.t.c.